docs/basics/data-format.md

```diff
@@ -17,13 +17,14 @@ finetuner.fit(model,
 
 This chapter introduces how to construct a `Document` in a way that Finetuner will accept.
 
-There are three different types of datasets:
+There are four different types of datasets:
 
 ```{toctree}
 :maxdepth: 1
 
 datasets/class-dataset
 datasets/session-dataset
+datasets/instance-dataset
 datasets/unlabeled-dataset
```

Comment:
where is this unlabeled dataset comes from?

Reply:
It's already there (check the documentation). It's basically only used in Labeler now